config/routes.rb

```diff
@@ -11,64 +12,11 @@
   namespace :api, defaults: {format: "json"} do
     namespace :v1 do
       get 'requests/show/:target(/:short_url(/:user_token))', to: "requests#show"
-      get 'requests/update/:target/:short_url/:user_token/:data', to: "requests#update"
-      get 'requests/destroy/:short_url/:user_token', to: "requests#destroy"
-      get 'requests/create/:original/:user_token(/:short_url)', to: "requests#create"
+      # get 'requests/update/:target/:short_url/:user_token/:data', to: "requests#update"
+      # get 'requests/destroy/:short_url/:user_token', to: "requests#destroy"
+      # get 'requests/create/:original/:user_token(/:short_url)', to: "requests#create"
     end
   end
-  get '/:path', to: "urls#reroute"
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
+  get '/:path', to: "reroute#index"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.